basics.js

```diff
@@ -4,19 +4,28 @@
 
 var pets = ['cat', 'dog', 'wombat'];
 
+for(var i = 0; i < pets.length; i ++){
```

Comment:
This starts off right, but is missing the inner content of the for loop. Also the incrementing `i++` shouldn't have a space between the `i` and the `++`.